app/ilmoweb/templates/open_labs.html

```diff
@@ -110,14 +112,29 @@ <h1 class="modal-title fs-5" id="studentsLabel">
              </div>
             </div>
            </td>
+          {% elif lab_group.signed_up_students == lab.max_students %}
+            <td>Täynnä</td>
           {% else %}
           <td>
             {% csrf_token %}
             <button class="btn btn-light btn-outline-dark" onclick="signup('{{ user.id }}', '{{ lab_group.id }}','{{ lab.name }}');">Ilmoittaudu</button>
           </td>
           {% endif %}
           {% if lab_group.status == 1 %}
-           <td>Ei vahvistettu</td>
+            {% if user.is_staff %}
+            <td>
+              {% if lab_group.signed_up_students >= lab.minim_students %}
```

Comment:
For now the if-clause is not needed when we don't want to have the minimum number of students set in models